docs/kubecon_outline.md

```diff
@@ -0,0 +1,29 @@
+# Streamlined Efficiency: Unshackling Kubernetes Image Volumes for Rapid AI Model and Dataset Loading
+
+## Part 1. Background and the questions raised (Esteban Rey)
+
+### Challenges in Managing Massive Data
+
```

Comment:
When relying on datasets in Kubernetes there are a number of challenges that any data store must be able to account for:
- Data must be highly available
- Data access must be performant
- Data access must be scalable

In addition to these, there are a number of challenges users will face when trying to manage such data:
- Datasets often require versioning
- Data can grow exponentially and determining what to cleanup can be daunting.